pkg/icingadb/v1/downtime.go

```diff
@@ -19,11 +19,13 @@ type Downtime struct {
 	EntryTime          types.UnixMilli `json:"entry_time"`
 	ScheduledStartTime types.UnixMilli `json:"scheduled_start_time"`
 	ScheduledEndTime   types.UnixMilli `json:"scheduled_end_time"`
-	FlexibleDuration   uint64          `json:"flexible_duration"`
+	ScheduledDuration  types.UnixMilli `json:"scheduled_duration"`
 	IsFlexible         types.Bool      `json:"is_flexible"`
+	FlexibleDuration   types.UnixMilli `json:"flexible_duration"`
 	IsInEffect         types.Bool      `json:"is_in_effect"`
 	StartTime          types.UnixMilli `json:"start_time"`
 	EndTime            types.UnixMilli `json:"end_time"`
+	Duration           types.UnixMilli `json:"duration"`
```

Comment:
This shouldn't use `UnixMilli` as that's a `time.Time` and thus represents a point in time. Either keep it as `uint64` or introduce something like `DurationMilli` that wraps a `time.Duration` and is represented as milliseconds in JSON/SQL.